### DISCUSSION TITLE
How to exclude navigation items from search?

### DISCUSSION THREAD

**Discussion by sccmcca** [squidfunk/mkdocs-material#4165] [General]
Use case: 

I have [`nav_translations`](https://github.com/ultrabug/mkdocs-static-i18n#nav_translations) configured, but have some instances where no translation is provided (i.e., defaulting onto my default language).

I want to exclude my translations from search. To accomplish this, I use the search [exclusion feature](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/?h=exclude#search-exclusion) on all my translated documents.

Here is the issue: My `nav_translations` are still searchable. I'd like to hide to these or all nav items from search. Is there any way to do that?

Thanks!

**Comment by squidfunk** [comment]
Are you using Insiders? Search exclusion is still a feature exclusive to sponsors.

**Reply by sccmcca** [reply]
Yes! I am using insiders.

**Reply by squidfunk** [reply]
Hmm, okay. I'm not sure how `nav_translations` is populated, since we don't officially support the `mkdocs-static-i18n` plugin. It might be related to an ordering issue in MkDocs event hooks, as both funtionalities are implemented as plugins. Please note that the `search` plugin __must__ be called last, and it appears that the `mkdocs-static-i18n` plugin [alters the search index](https://github.com/ultrabug/mkdocs-static-i18n/blob/e188e9822a716b1e2c18b2821935512b9d485336/mkdocs_static_i18n/plugin.py#L399).

Please follow up with the maintainers of this plugin. Essentially, what search exclusion does is to skip certain pages, sections and paragraphs when building the search index. This is implemented as part of the built-in search plugin which is bundled with Insiders.
